src/node/rpc/member_frontend.h

```diff
@@ -514,6 +514,8 @@ namespace ccf
       add_kv_wrapper_endpoint(network.constitution);
 
       add_kv_wrapper_endpoint(ccf::Service(ccf::Tables::SERVICE));
```

Comment:
It looks like the [SNP tables](https://microsoft.github.io/CCF/main/audit/builtin_maps.html#nodes-snp-host-data) should also be covered here, as the check is failing on SNP: https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=56937&view=logs&j=b450cd14-f66d-5f5f-6465-8f7fbd24a6b4&t=b7087068-73db-5a59-8152-5a000f437c3b&l=20635. 

I also feel like we could remove the existing `GET /snp/host_data` and `GET /snp/measurements` endpoints (`common_endpoint_registry.cpp`) as they'd be superseded by the new generic endpoints and they haven't been used by customers yet? @DomAyre, what do you think?

Reply:
Tracking this in #4687.